test/kuttl/tests/glance_disk_formats/01-create_image.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+    - command: |
```

Comment:
-1:
```
logger.go:42: 11:46:39 | glance_disk_formats/0-deploy | test step failed 0-deploy
case.go:378: failed in step 0-deploy
case.go:380: pods "openstackclient" not found
```

As done for image_cache and dist image import, you need to update the openstackclient configmap to get the script that is run during the test (and deploy the pod).